Doc/whatsnew/3.12.rst

```diff
@@ -1314,46 +1326,64 @@ Removed
 
   (Contributed by Pradyun Gedam in :gh:`95299`.)
 
-* :mod:`enum`: Remove ``EnumMeta.__getattr__``, which is no longer needed for
+enum
+----
+
+* Remove :mod:`enum`'s ``EnumMeta.__getattr__``, which is no longer needed for
   enum attribute access.
   (Contributed by Ethan Furman in :gh:`95083`.)
 
-* :mod:`ftplib`: Remove the ``FTP_TLS.ssl_version`` class attribute: use the
+ftplib
+------
```

Comment:
By moving bullets to subheads we loose the module links -- should we add them back? (Happy to tackle this in a follow-up PR if this is agreed)

Reply:
The module link is useful but I think we should avoid repeating them:

---

## ftplib

* ftplib: Remove the ``FTP_TLS.ssl_version`` class attribute: use the *context* parameter instead.

---

I had generally slurped the module link into the main text (e.g. enum), when there wasn't one already there (e.g. importlib).

Looks like I missed one or two, let me check and add.

Reply:
Updated, how's that?

Reply:
Fab, thanks